control-plane/csi-driver/src/bin/node/mount_utils/bin/mount_utils.rs

```diff
@@ -0,0 +1,50 @@
+use crate::error::MountError;
+
+use sys_mount::{FilesystemType, Mount, MountFlags, UnmountFlags};
+
+/// Calls sys mount's mount in a spawn blocking.
+pub(crate) async fn mount(
+    device_path: String,
+    target: String,
+    data: Option<String>,
+    mount_flags: MountFlags,
+    fstype: Option<String>,
+) -> Result<(), MountError> {
+    let blocking_task = tokio::task::spawn_blocking(move || {
+        let mut mount = Mount::builder();
+
+        if let Some(data_str) = data.as_deref().filter(|d| !d.is_empty()) {
+            mount = mount.data(data_str);
+        }
+
+        mount
```

Comment:
wonder if we need this at all anymore, if it's not safe to call ioctl directly, maybe we can simply call the mount binary? Though it might be safer to do this first until we figure out how to migrate from this to mount/unmount arguments properly.

Reply:
With using binaries directly, the errors become a bit vague, but anyhow yeah we can do that. For now kept it as it was to keep the same behaviour.